students/shawn/class_03/slicing_lab.py

```diff
@@ -39,11 +39,11 @@ def seq_rev(seq):
 
 # print(seq_rev('abcdefg'))
 
-import  math as m
+
```

Comment:
Can you do this using just list slicing, considering directionality? For example, to take the last element in a list, you can use seq[-1:]